<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_win</class>
 <widget class="QWidget" name="Admin_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin_win</string>
  </property>
  <widget class="QTabWidget" name="TabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #EBE9F5;</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="Info_tab">
    <property name="styleSheet">
     <string notr="true">#Info_tab{border-image: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/purple_background.png)}</string>
    </property>
    <attribute name="title">
     <string>Info</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>611</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #EBE9F5;
border-radius: 25px;
</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>291</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FFFFFF;
border-radius: 25px;
</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7D43E2</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>140</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7D43E2</string>
       </property>
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
      <widget class="QLabel" name="category_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="surname_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Admin Info</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="members_tab">
    <attribute name="title">
     <string>Members</string>
    </attribute>
    <widget class="QTableWidget" name="members_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>981</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #FFFFFF;
selection-background-color: #5492dc;
border: 1px solid;
border-color: #FFFFFF;
color: #080c2b;
border-radius:15px;
padding: 30px;
background: #FFFFFF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>surname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Erase</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data_update</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gym members</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button_mb">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>50</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
background-color: #7D43E2;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="members_csv_button">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Get csv file</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="trainers_tab">
    <attribute name="title">
     <string>Trainers</string>
    </attribute>
    <widget class="QTableWidget" name="trainers_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>981</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #FFFFFF;
selection-background-color: #5492dc;
border: 1px solid;
border-color: #FFFFFF;
color: #080c2b;
border-radius:15px;
padding: 30px;
background: #FFFFFF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>180</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>surname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delete</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gym trainers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainers_csv_button">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Get csv file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button_tr">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>50</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
background-color: #7D43E2;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sessions_tab">
    <attribute name="title">
     <string>Sessions</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sessions</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="sessions_table">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>841</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #FFFFFF;
selection-background-color: #5492dc;
border: 1px solid;
border-color: #f3f3f3;
color: #080c2b;
border-radius:15px;
padding: 30px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>170</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Slot</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>About</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Member</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="sessions_csv_button">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Get csv file</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
